src/html/html-script-scanner.ts

```diff
@@ -43,16 +43,16 @@ export class HtmlScriptScanner implements HtmlScanner {
         if (src) {
           const importUrl = resolveUrl(document.url, src);
           features.push(new ScannedScriptTagImport(
-              'html-script', importUrl, document.sourceRangeForNode(node),
-              document.sourceRangeForAttribute(node, 'src'), node));
+              'html-script', importUrl, document.sourceRangeForNode(node)!,
+              document.sourceRangeForAttribute(node, 'src')!, node));
         } else {
           const locationOffset = getLocationOffsetOfStartOfTextContent(node);
-          const attachedCommentText = getAttachedCommentText(node);
+          const attachedCommentText = getAttachedCommentText(node) || '';
```

Comment:
same here and for more below, if there's no attached comment text can we leave as undefined?